src/uploads/upload.entity.ts

```diff
@@ -64,6 +64,14 @@ export class Upload {
   @Column({ name: 'github_job' })
   githubJob!: string;
 
+  // The URL of a GitHub REST API endpoint describing the job that performed this upload.
+  @Column({ name: 'github_job_api_url', nullable: true, type: 'varchar' })
+  githubJobApiUrl!: string | null;
```

Comment:
`githubJobApiUrl` - is this one added for future-proofing? It's not visible right now in the UI and not used in the code

Reply:
> is this one added for future-proofing?

Yes, that's right. Seemed like a useful piece of data to keep around.